CUEstion.WEB/Controllers/QuestionsController.cs

```diff
@@ -217,13 +209,12 @@ public IActionResult VoteForAnswer(int questionId, int answerId)
 
 
 
-		[HttpOptions("{questionId:int}/comments/{commentId:int}")]
+		[HttpPut("{questionId}/comments/{commentId}")]
 		[Authorize]
 		public IActionResult VoteForCommentToQuestion(int questionId, int commentId)
```

Comment:
For all these methods which handle "mark" you can add **mark** argument to method signatures